cluster/addons/podsecuritypolicies/kube-system-role.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+kind: ClusterRole
+metadata:
+  labels:
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
+  name: kube-system:default:podsecuritypolicy
```

Comment:
I'd expect the role name to reflect the name of the psp it was granting access to. Also, nothing about this role ties it to the kube-system namespace. Maybe something like `podsecuritypolicy:privileged`?